app/Handlers/GithubWebhookHandler.php

```diff
@@ -0,0 +1,63 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Handlers;
+
+use App\Services\OrganizationService;
+use Illuminate\Http\Request;
+use Illuminate\Support\Facades\Log;
+use Illuminate\Validation\UnauthorizedException;
+use Symfony\Component\HttpKernel\Exception\BadRequestHttpException;
+
+class GithubWebhookHandler
+{
+    public function handle(Request $request): void
+    {
+        Log::info("Received webhook request:", $request->all());
+
+        $this->validateGithubWebhook(config("services.github.webhook_secret"), $request);
+
+        Log::info($request->getContent());
+
+        $organizationService = new OrganizationService();
+
+        switch ($this->getActionType($request)) {
+            case "created":
+                $organizationService->create(data_get($request, "installation.account"));
```

Comment:
We can prepare that data into some DTO or smth like that?

Reply:
I added organization DTO